### PR TITLE
Code Review: Fix Artboard.adjustToFit in SketchAPI (#17202)

### LINKED ISSUE #111: adjustToFit method not work for Artboard
It looks like the methods inherits from `Group`, but it just not works.
I find it works by following script
```javascipt
document.sketchObject.actionsController().actionForID("MSResizeArtboardToFitAction").resizeArtboardToFit(nil)
```
but should ensure the `artboard` layer is selected..

Did I miss something or is this a bug? Thanks!

### PR DESCRIPTION
**Pull request by mathieudutour**
Connect BohemianCoding/Sketch#17202

Fix #111

### CODE REVIEW

File: Source/dom/components/Artboard.js [hunk 1]
```diff
@@ -24,6 +24,15 @@ export class Artboard extends Group {
     }
     super(artboard)
   }
+
+  /**
+   * Adjust the Artboard to fit its children.
+   * override the group's method
+   */
+  adjustToFit() {
+    this._object.resizeToFitChildren()
```
Comment (nichenqin) [inline Source/dom/components/Artboard.js]: is it `resizeToFitChildrenWithOption`?

Comment (mathieudutour) [reply Source/dom/components/Artboard.js]: no, `resizeToFitChildrenWithOption` doesn't do anything as you notice. This is a new method that I created

Comment (nichenqin) [reply Source/dom/components/Artboard.js]: this method will not be available until v49.1?

Comment (mathieudutour) [reply Source/dom/components/Artboard.js]: no I just created it :)

**Pull request merged**